src/main/java/appointment/watcher/Watcher.java

```diff
@@ -18,35 +27,35 @@ public static void main(String[] args) throws Exception {
                 //TODO: send text only when appointment is available
                 available = watcherService.checkIfAppointmentAvailable();
                 sendText("Appointment available = " + available);
-            } catch (Exception e) {
-                System.out.println(e.getCause());
-                sendText("Something went wrong=" + e.getMessage());
+            } catch (CustomException e) {
```

Comment:
We can catch very specific exception? to convey what is CustomException is about?